demo/index.md

```diff
@@ -205,6 +205,67 @@ It is possible to automatically sort the table once you create a Tablesort insta
 </tbody>
 </table>
 
+### Sorting by column keys
+
+Sometimes, tables can have more complex column structures, especially when using colspans. In these cases, you can
+explicitly connect a header to the cells in each row that it should sort by, by using the `data-sort-column-key`
+attribute.
+
+This example sorts products by price, even though the prices are not in the same column as their header.
+
+{% highlight html %}<table id='column-keys' class='sort'>
+<thead>
+  <tr>
+    <th >Product</th>
+    <th colspan=2 data-sort-column-key="price">Price</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Apples</td>
+    <td>Sale!</td>
+    <td>20</td>
+  </tr>
+  <tr>
+    <td>Bread</td>
+    <td>Out of stock</td>
+    <td>10</td>
+  </tr>
+  <tr>
+    <td>Radishes</td>
+    <td>In Stock!</td>
+    <td>30</td>
+  </tr>
+</tbody>
+</table>
+{% endhighlight %}
+
+<table id='column-keys' class='sort'>
```

Comment:
I know the documentation of this project is a little horrendous but for your code example to work, can you add 

```js
new Tablesort(document.getElementById('column-keys'));
``` 

Within the `<script>` tag [here](https://github.com/tristen/tablesort/pull/181/files#diff-46c1f60845b5267148ebc4d1a8f62654R282)?

Reply:
Ah, I incorrectly assumed that the script at the bottom would initiate `Tablesort`s on all `table.sort` elements in here. My mistake!

I've given your suggestion a whirl!